src/System.Device.Gpio/Interop/Interop.OperatingSystem.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal static class OperatingSystem
```

Comment:
Today we compile two different versions of System.Device.Gpio, one for Windows and one for Linux, which is why there is no need yet for these methods. Instead, what we prefer is to have one method return a I2c device for linux, and another one that returns a I2c device for windows.